netbout-client/src/main/java/com/netbout/mock/MkFriends.java

```diff
@@ -78,9 +78,11 @@
     public void invite(final String friend) throws IOException {
         try {
             new JdbcSession(this.sql.source())
-                .sql("INSERT INTO friend (bout, alias) VALUES (?, ?)")
+                // @checkstyle LineLength (1 line)
+                .sql("INSERT INTO friend (bout, alias, subscription) VALUES (?, ?, ?)")
                 .set(this.bout)
                 .set(friend)
+                .set(1)
```

Comment:
@longtimeago It means True, subscription is enabled for target friend with regard to current bout